num_traj: 0

use_planner: true

MPC:
  N: 15
  dt: 0.5
  SQP_iters: 4
  stateScaling: [1, 1, 1, 1]
  inputScaling: [.1, .1]
  terminalScaling: 10
  tau_max: 5
  vel_max: 0.2
  use_previous_reference: true

Planner:
  buffer: 0.2
  num_points: 1500
  x_bounds: [-1.5, 1.5, -1, 1] # min_x, max_x, min_y, max_y
  dx_bounds: [0, 0, 0, 0] # min_dx, max_dx, min_dy, max_dy
  log_edges: false